test/llvm_ir_correct/tbaa_multimod_01_input.f90

```diff
@@ -0,0 +1,25 @@
+! A part of the test provided in tbaa_multimod_01.f90
+! Empty RUN and CHECK to prevent error
+! RUN: echo "NoCheck" | FileCheck %s
+! CHECK: NoCheck
```

Comment:
Is it possible to move these files into `test/llvm_ir_correct/Inputs/`, and remove the unnecessary `RUN` and `CHECK` directives?